src/zenml/utils/code_repository_utils.py

```diff
@@ -125,6 +128,12 @@ def find_active_code_repository(
         local_context = repo.get_local_context(path)
         if local_context:
             break
+    else:
+        if code_repositories and log:
+            # There are registered code repositories, but none was matching the
+            # current path -> We log the path to help in debugging issues
+            # related to the source root.
+            logger.info("No matching code repository found at path %s.", path)
```

Comment:
shouldnt we also log when it does find it?

Reply:
That already happens in a different place